src/pages/Login/index.scss

```diff
@@ -0,0 +1,12 @@
+.login-wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+  background: #f5f5f5;
+
+  .card {
+    width: 350px;
```

Comment:
Just Add `margin-top: auto` here It will fix this problem for 13.3-inch laptops as well!